src/blocks/ContentLayout/ContentLayout.scss

```diff
@@ -60,6 +60,12 @@ $block: '.#{$ns}content-layout-block';
         padding: $indentXL;
     }
 
+    &_theme {
+        &_dark {
+            --g-color-line-focus: var(--g-color-text-light-primary);
```

Comment:
What color is default for `--g-color-line-focus` in uikit and why can't we use ut?

Reply:
Ok, let's use default focus color in page-constructor